Stack/Opc.Ua.Core/Stack/Server/EndpointBase.cs

```diff
@@ -611,6 +612,9 @@ protected class ServiceDefinition
             /// </summary>
             /// <param name="requestType">Type of the request.</param>
             /// <param name="invokeMethod">The invoke method.</param>
+#if NET_STANDARD_OBSOLETE_SYNC && false
```

Comment:
Remove the #if / #endif.  We should just obsolete this now.  NET_STD_OBSOLETE will always be set, and no one should expect it to work without it.  Same as with the other ones in this file.  Lets remove as many #if in the code as possible, then we can better understand the support surface.